run-tests.xsh

```diff
@@ -26,26 +37,23 @@ def test(ns: argparse.Namespace):
 
     args = ns.pytest_args
 
-    if not $(xonsh -c "import xonsh.main; print(xonsh.main.__file__, end='')").endswith("__amalgam__.py"):
-        echo "Tests need to run from the amalgamated xonsh! install with `pip install .` (without `-e`)"
-        exit(1)
+    # if not $(xonsh -c "import xonsh.main; print(xonsh.main.__file__, end='')").endswith("__amalgam__.py"):
+    #     echo "Tests need to run from the amalgamated xonsh! install with `pip install .` (without `-e`)"
+    #     exit(1)
 
+    cmd = ["pytest"] + _replace_args(args, 0)
     if ns.report_coverage:
-        ![pytest @(_replace_args(args, 0)) --cov --cov-report=xml --cov-report=term]
-    else:
-        ![pytest @(_replace_args(args, 0))]
+        cmd += "--cov --cov-report=xml --cov-report=term".split()
+    subprocess.check_call(cmd, stderr=sys.stderr, stdout=sys.stdout)
```

Comment:
Using xonsh here would cause this environment variables to get passed onto integration tests and they will fail. 
One other thing, it uses pytest-timeout on windows which causes slowdown on windows. We can use the subprocess timeout here

Reply:
seems like removing pytest-timeout doesn't reduce the total time on windows. reverted the change